vassal-app/src/main/java/VASSAL/counters/PieceDefiner.java

```diff
@@ -175,13 +177,7 @@ public void setPiece(GamePiece piece) {
       piece = ((Decorator) piece).piece;
     }
 
-    if (piece == null) {
-      inUseModel.insertElementAt(new BasicPiece(), 0);
-    }
-    else {
-      inUseModel.insertElementAt(piece, 0);
-    }
-
+    inUseModel.insertElementAt(Objects.requireNonNullElseGet(piece, BasicPiece::new), 0);
```

Comment:
Isn't `piece == null ? new BasicPiece() : piece` both shorter and easier to read?

Reply:
My IDE suggests this "modern" way of doing this but either way works for me. This is probably meant to be use with a static import of the method, then it would just be `model.insertElementAt(requireNonNullElseGet(piece, BasicPiece::new), 0)`